pkg/provisioner/templates/kubernetes.go

```diff
@@ -95,10 +95,10 @@ export KUBECONFIG="${HOME}/.kube/config"
 
 # Install Calico
 # based on https://docs.tigera.io/calico/latest/getting-started/kubernetes/quickstart
-with_retry 3 10s kubectl --kubeconfig $KUBECONFIG apply -f https://raw.githubusercontent.com/projectcalico/calico/${CALICO_VERSION}/manifests/tigera-operator.yaml
+with_retry 3 10s kubectl --kubeconfig $KUBECONFIG create -f https://raw.githubusercontent.com/projectcalico/calico/${CALICO_VERSION}/manifests/tigera-operator.yaml
```

Comment:
Oh I see [why](https://github.com/projectcalico/calico/issues/7826#issuecomment-1614170418) now. This is unfortunate. The problem with `create` is that it leads to flaking errors

Reply:
We need to address this change separately in another PR. 

Let's keep this PR focused on the metadata addition, please

Reply:
Do you mean I should raise a separate PR for kubectl create, and after merging the kubectl create PR we should push this PR ? 

as without _these changes it may give the above error.

Reply:
It is not clear to me why this change is necessary for this PR. Wasn't `kubectl apply` working before?

Reply:
@cdesiniotis I created a new [PR](https://github.com/NVIDIA/holodeck/pull/243). I've added details there